.travis.yml

```diff
@@ -2,8 +2,15 @@ language: node_js
 node_js:
   - "stable"
 sudo: false
+env: REACT_VERSION=15.3.x
 matrix:
   include:
-    - env: REACT=14.x
-    - env: REACT=15.x
+    # These versions run different jobs in which we `yarn add` other releases
+    # of react (& friends) to run our test suite against those as well
+    - env: REACT_VERSION=0.14.x
 cache: yarn
+install:
+  - yarn
+  - yarn add react@$REACT_VERSION
+  - yarn add react-dom@$REACT_VERSION
+  - yarn add react-addons-test-utils@$REACT_VERSION
```

Comment:
Multi line `yarn add` statements like this make this appear to run the tests every time after the install. These should be collapsed into 1 statement, or perhaps this is occurring due to not having something like `in-publish` configured.